_unittest/test_07_Object3D.py

```diff
@@ -537,6 +537,11 @@ def test_26_unclassified_object(self):
         self.aedtapp.modeler.oeditor.Intersect(vArg1, vArg2)
         assert box1 in self.aedtapp.modeler.unclassified_objects
 
+    def test_26a_delete_unclassified_object(self):
+        unclassified = self.aedtapp.modeler.unclassified_objects
+        assert self.aedtapp.modeler.delete(unclassified)
+        assert len(self.aedtapp.modeler.unclassified_objects) != unclassified
```

Comment:
Shouldn't you test that the value is equal to 0 since the got deleted ?

Reply:
@SMoraisAnsys yes I could but it's the same at the end. I can add a new line.